circuit_knitting/cutting/cut_finding/cut_optimization.py

```diff
@@ -125,7 +130,7 @@ def cut_optimization_goal_state_func(
 # Global variable that holds the search-space functions for generating
 # the cut optimization search space.
 cut_optimization_search_funcs = SearchFunctions(
-    cost_func=cut_optimization_cost_func,
+    cost_func=cut_optimization_upper_bound_cost_func,  # valid choice when considering only LO cuts.
```

Comment:
How does this line changed relate to the change in #585?

Reply:
It's somewhat orthogonal but I think this is what (for consistency) it ought to have been from the start. It's hiding all LOCC related cost tracking.

Reply:
Should this change be backported too?  What are the consequences of the stable release lacking this update?  (Are there any?)

Reply:
I don't think there are any consequences because the part that this effects was not exposed through our API. It's more just for the sake of consistency. I think we should be okay to not backport it.

Reply:
Great; I thought that might be the case, just wanted to confirm :slightly_smiling_face:

Reply:
Always a good idea!